js/components/multi-audio/multiAudioPopover.js

```diff
@@ -38,11 +41,17 @@ var MultiAudioPopover = React.createClass({
             closeAction={this.handleClose} />
         </li>
         {multiAudio.map(function(audio){
+          var color = "white";
+          if (audio.enabled) {
+            color = "blue";
```

Comment:
Should we have this color be configurable?

Reply:
@aeng7 this will be implemented in another pull requests. For such customization we have the following ticket - https://jira.corp.ooyala.com/browse/PLAYER-543